math/foam/inc/TFoam.h

```diff
@@ -47,7 +46,7 @@ class TFoam : public TObject {
 
    TFoamMaxwt   *fMCMonit;    ///< Monitor of the MC weight for measuring MC efficiency
    Double_t   fMaxWtRej;      ///< Maximum weight in rejection for getting wt=1 events
-   TRefArray *fCellsAct;      ///< Array of pointers to active cells, constructed at the end of foam build-up
+   std::vector<Long_t> fCellsAct; ///< Index of active cells, constructed at the end of foam build-up
```

Comment:
Since now we are using an std::vector, should not we add an #include <vector>

Reply:
Oh, good idea, indeed.